openedx/core/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -238,6 +277,40 @@ def test_dop_public_client_access_token(self):
         data = json.loads(response.content)
         self.assertNotIn('refresh_token', data)
 
+    @ddt.data(dot_models.Application.GRANT_CLIENT_CREDENTIALS, dot_models.Application.GRANT_PASSWORD)
+    def test_jwt_access_token_scopes_and_filters(self, grant_type):
+        """
+        Verify the JWT contains the expected scopes and filters.
+        """
+        dot_app = self.dot_adapter.create_public_client(
+            name='test dot application',
+            user=self.user,
+            redirect_uri=DUMMY_REDIRECT_URL,
+            client_id='dot-app-client-id-{grant_type}'.format(grant_type=grant_type),
+            grant_type=grant_type,
+        )
+        dot_app_access = models.ApplicationAccess.objects.create(
+            application=dot_app,
+            scopes=['grades:read'],
+        )
+        dot_app_org = models.ApplicationOrganization.objects.create(
+            application=dot_app,
+            organization=OrganizationFactory()
+        )
+        scopes = dot_app_access.scopes
+        filters = [dot_app_org.to_jwt_filter_claim()]
+        if grant_type != dot_models.Application.GRANT_CLIENT_CREDENTIALS:
+            filters.append(DOTAdapter.FILTER_USER_ME)
```

Comment:
Can you call `get_authorization_filters` instead?  So you don't have to add the `FILTER_USER_ME` filter explicitly and can test that it's done by the application?